nbconvert/preprocessors/execute.py

```diff
@@ -45,7 +45,7 @@ class ExecutePreprocessor(Preprocessor):
             If a cell execution takes longer, an exception (TimeoutError
             on python 3+, RuntimeError on python 2) is raised.
 
-            `None`, or `0` will disable the timeout.
+            `None` or `-1` will disable the timeout.
```

Comment:
I'm matching the style of other helps in this file:

```

    allow_errors = Bool(
        False, config=True,
        help=dedent(
            """
            If `False` (default), when a cell raises an error the
            execution is stoppped and a `CellExecutionError`
            is raised.
            If `True`, execution errors are ignored and the execution
            is continued until the end of the notebook. Output from
            exceptions is included in the cell output in both cases.
            """
        )
    )

```

I dont 'care if we decorate or not. I can double them all.